src/components/Routes/index.tsx

```diff
@@ -1,6 +1,11 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Does this happen in more than one place of only in the `useEffect` hook at line 34? Ideally you should include the values you're referring inside the hook in the dependency array, but if for some reason you need to ignore those deps, disable the eslint rule for that particular line only (it helps us better understand what you're disabling).